packages/client/src/api/leaderboardAPI.ts

```diff
@@ -0,0 +1,19 @@
+import {api} from './request';
+import {
+	TLeaderboard, TAddLeaderboard, TLeaderboardResponse,
+} from './typing';
+import {TEAM_NAME} from './constants';
+
+export const leaderboardAPI = {
+
+	getAllLeader: (data: TLeaderboard) => (
+		api.post<Array<TLeaderboardResponse>>('leaderboard/all', data)
+	),
+	addUserToLeaderboard: (data: TAddLeaderboard) => (
```

Comment:
Будь консистентым (повторяющимся)

```suggestion
	),
  
	addUserToLeaderboard: (data: TAddLeaderboard) => (
```